pandas/core/groupby/groupby.py

```diff
@@ -190,14 +190,14 @@ class providing the base-class of operations.
 )
 
 _pipe_template = """
-Apply a function `func` with arguments to this %(klass)s object and return
-the function's result.
+Apply a function `func` with arguments to this %(klass)s object and return the result.
 
 %(versionadded)s
 
+Instead of writing:
```

Comment:
Can you restore this? I think it made more sense where it was. Looks out of context here.